app/lib/features/pins/widgets/pin_list_item.dart

```diff
@@ -33,7 +33,7 @@ class PinListItemById extends ConsumerWidget {
         showSpace: showSpace,
       ),
       error: (err, st) => Text(
-        '${L10n.of(context).error} ${L10n.of(context).loading('').toLowerCase()}: ${err.toString()}',
+        '${L10n.of(context).error} ${L10n.of(context).loading.toLowerCase()}: ${err.toString()}',
```

Comment:
why isn't this the same `failedToLoad(error)` as used elsewhere?

```suggestion
L10n.of(context).failedToLoad(err.toString())
```